cmdrs-guide.html

```diff
@@ -0,0 +1,137 @@
+        <p>
+            Like all great questions, the answer is; it depends. There are broadly 2 types of tools:
+        </p>
+        <ol>
+            <li>
+                Client tools, running on your PC.
+            </li>
+            <li>
+                Browser based tools, running on some remote server and accessed using your web browser.
+            </li>
+        </ol>
+        <p>
+            Client tools would need to be downloaded and run from your system. Most of the tools need to be run on the
+            same system that Elite Dangerous is running on although some tools might not need so.
+        </p>
+        <p>
+            The available tools can be used for anything from tracking and logging your activities, keeping track of
+            your resources and credits, to giving advice for how to build that perfect ship you are looking for.
+        </p>
+        <h3>How do these tools get their data?</h3>
+        <p>
+            There are 3 ways tools get their data:
+        </p>
+        <ol>
+            <li>
+                From journal files generated by Elite Dangerous. Some use the data of these files sent through EDDN.
+            </li>
+            <li>
+                From CAPI once you authenticate your Frontier account with the tools.
+            </li>
+            <li>
+                From manual user input
+            </li>
+        </ol>
+        <h3>Whoa! What are journal files, EDDN and CAPI?</h3>
+        <h4>Journal Files</h4>
+        <p>
+            When you start Elite Dangerous and do activities in it, the game writes some particular files in the system
+            you are playing on. These files keep track of your activities by writing
+            <a href="https://www.w3schools.com/whatis/whatis_json.asp">JSON</a> data. These files are called the Journal
+            files. By default you can find these files at
+            <code>C:\Users\%userprofile%\Saved Games\Frontier Developments\Elite Dangerous</code> (Please note that
+            depending on your setup, the location might be different).
+        </p>
+        <p>
+            The files in this folder can track back years if you haven't changed your system. That means these files do
+            not carry over when you switch PCs. One way to carry them over is to manually copy from the ols system and
+            paste them in the new system.
+        </p>
+        <h4>Elite Dangerous Data Network (EDDN)</h4>
+        <p>
+            Elite Dangerous Data Network or EDDN is a community run message broker used by browser based tools. Client
+            tools send data to EDDN and browser based tools get the data from EDDN. EDDN itself doesn't store any data.
+            It just passes along data.
+        </p>
+        <h4>Commander API (CAPI)</h4>
+        <p>
+            The commander API or the CAPI is an <a href="https://en.wikipedia.org/wiki/API">API</a> provided by the game
```

Comment:
'CAPI' is 'Companion API' - so named from the long-since defunct iOS 'Elite Dangerous Companion' app.

Reply:
🤦🏼‍♂️